README.md

```diff
@@ -93,7 +93,7 @@ thelounge --help
 The following commands install and run the development version of The Lounge:
 
 ```sh
-git clone https://github.com/thelounge/lounge.git
+git clone https://github.com/thelounge/thelounge.git
 cd lounge
```

Comment:
`cd thelounge` :)

Reply:
Actually, nevermind, I'm amending your commit with:

```diff
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: README.md
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
@ README.md:97 @ The following commands install and run the development version of The Lounge:

 ```sh
 git clone https://github.com/thelounge/thelounge.git
-cd lounge
+cd thelounge
 yarn install
 NODE_ENV=production yarn build
 yarn start
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: scripts/changelog.js
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
@ changelog.js:645 @ async function generateChangelogEntry(targetVersion) {
    let template;
    let contributors = [];

-   const codeRepo = new RepositoryFetcher(client, "lounge");
+   const codeRepo = new RepositoryFetcher(client, "thelounge");
    const previousVersion = await codeRepo.fetchPreviousVersion(targetVersion);

    if (isPrerelease(targetVersion)) {
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: src/models/user.js
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
@ user.js:20 @ function User(attr, prefixLookup) {
 }

 User.prototype.setModes = function(modes, prefixLookup) {
-   // irc-framework sets character mode, but lounge works with symbols
+   // irc-framework sets character mode, but The Lounge works with symbols
    this.modes = modes.map((mode) => prefixLookup[mode]);

    this.mode = this.modes[0] || "";
```